src/assets/i18n/fr.json

```diff
@@ -1074,13 +1069,12 @@
   "retreat.form.type.choices.physical": "Retraite en présentiel",
   "retreat.form.type.choices.virtual": "Retraite virtuelle",
   "retreat.form.videoconference_link": "Lien de la vidéoconference",
-  "retreat.form.videoconference_tool": "Outil de vidéoconference",
+  "retreat.form.videoconference_tool": "Outil de videéconference",
```

Comment:
You hadded a typo here, last version was good. Maybe just a keyboard fail ?

Reply:
vidéoconference fixed